doc/users/project/mission.rst

```diff
@@ -0,0 +1,27 @@
+Matplotlib Mission Statement
+============================
+
+The Matplotlib developer community develops, maintains, and
+supports Matplotlib and its extensions, providing plotting tools for 
+the Scientific Python Ecosystem.  Our success is based on how effectively we
+enable our users, across scientific, industrial, commercial, and educational
+applications, to achieve their goals.  We believe that a diverse developer community creates
+the best software, and we welcome anyone, regardless of experience, education,
+gender, or nationality, who shares our mission to the community.
```

Comment:
I worry explicitly listing some groups but not others - like race/ethnicity, religion/creed,  sexual orientation - might get us into "why those?" discussions. Maybe go more general here?

```suggestion
applications, to achieve their goals.  We believe that a diverse developer community creates
the best software, and we welcome anyone who shares our values, as described in the code of conduct, and our mission to the community.
```